static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx

```diff
@@ -0,0 +1,138 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import OnboardingIntegrationSection from 'sentry/components/events/featureFlags/onboardingIntegrationSection';
+import {AuthTokenGeneratorProvider} from 'sentry/components/onboarding/gettingStartedDoc/authTokenGenerator';
+import type {OnboardingLayoutProps} from 'sentry/components/onboarding/gettingStartedDoc/onboardingLayout';
+import {Step, StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import type {DocsParams} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {useSourcePackageRegistries} from 'sentry/components/onboarding/gettingStartedDoc/useSourcePackageRegistries';
+import {useUrlPlatformOptions} from 'sentry/components/onboarding/platformOptionsControl';
+import ConfigStore from 'sentry/stores/configStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {space} from 'sentry/styles/space';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export function FeatureFlagOnboardingLayout({
+  docsConfig,
+  dsn,
+  platformKey,
+  projectId,
+  projectSlug,
+  newOrg,
+  projectKeyId,
+  configType = 'onboarding',
+  integration = '',
+  provider = '',
+}: OnboardingLayoutProps) {
+  const api = useApi();
+  const organization = useOrganization();
+  const {isPending: isLoadingRegistry, data: registryData} =
+    useSourcePackageRegistries(organization);
+  const selectedOptions = useUrlPlatformOptions(docsConfig.platformOptions);
+  const {isSelfHosted, urlPrefix} = useLegacyStore(ConfigStore);
+
+  const {introduction, steps} = useMemo(() => {
+    const doc = docsConfig[configType] ?? docsConfig.onboarding;
```

Comment:
Does it make sense to have use default docs as fallback here?

Reply:
i don't know what we would use instead😅